lib/Language/en/translation.json

```diff
@@ -275,7 +275,7 @@
   },
   "welcomeMessage": {
     "title": "Let's get you started",
-    "welcomeMessage": "Interested in data discovery and exploration?<br/>Dive right in and get started or check the following help guide options.",
+    "welcomeMessage": "Welcome to the Queensland Digital Twin.<br/> Access a rich catalogue of 2D, 3D and live data from multiple sources in one convenient location. Take a tour guide or start exploring!",
```

Comment:
Revert this change, the QLD terriamap translation script will override this so just watch when you are staging files

Reply:
I was puzzeld by this changes. Now clear.

Reply:
done